Psychtoolbox/PsychHardware/EyelinkToolbox/EyelinkBasic/Contents.m

```diff
@@ -0,0 +1,7 @@
+% EyelinkToolbox:EyelinkBasic
+% collection of essential functions for the eyelink toolbox
+%
+% EyelinkBasicLegacy - Deprecated functionality retained for warnings
```

Comment:
I'm not sure if I'm confused here about what you changed, but the idea is not to replace functions with functions that **just output a warning**, but to **replace them with the old legacy functions** and **additionally** add a warning. Otherwise you are still breaking old user scripts, just now with a warning.

Reply:
resolved in 33984c0ef6fc9d2a3dc40f30ab9710ad031d9938